### PR TITLE
Yann/cassandra metric mapping update

### PR DESCRIPTION
**Pull request by yannmh**
Rebase of https://github.com/DataDog/jmxfetch/pull/50

**Added changes**
Create a `cassandra_aliasing` flag to select the appropriate aliasing logic.
* `cassandra_aliasing` → compliant with CASSANDRA-4009
* default behavior

### CODE REVIEW

**Comment by yannmh** [comment]
@remh can you take a quick pass on the last commit please ?

File: src/main/java/org/datadog/jmxfetch/reporter/Reporter.java [hunk 1]
```diff
@@ -60,8 +60,6 @@ public void sendMetrics(LinkedList<HashMap<String, Object>> metrics, String inst
             // We need to edit metrics for legacy reasons (rename metrics, etc)
             HashMap<String, Object> metric = new HashMap<String, Object>(m);
 
-            postProcess(metric);
```
Comment (olivielpeau) [inline src/main/java/org/datadog/jmxfetch/reporter/Reporter.java]: You can also remove this method's definition, along with `postProcessCassandra` that is not used anymore.

File: src/main/java/org/datadog/jmxfetch/JMXSimpleAttribute.java [hunk 2]
```diff
@@ -90,17 +91,34 @@ private String getAlias() {
...
+            alias = "jmx." + getDomain() + "." + getAttributeName();
         }
         alias = convertMetricName(alias);
         return alias;
     }
 
+    private String getCassandraAlias() {
+        if (renameCassandraMetrics()) {
+            Map<String, String> beanParameters = getBeanParameters();
+            String type = beanParameters.get("type");
```
Comment (olivielpeau) [inline src/main/java/org/datadog/jmxfetch/JMXSimpleAttribute.java]: This variable is never used.

**Comment by yannmh** [comment]
Thanks @olivielpeau. 
I addressed your comments and squashed the commits. Should be ready to merge.

**Comment by yannmh** [comment]
Any thought on the `cassandra_aliasing` parameter name, before this get merged ?  I am not entirely satisfied by the name, but I can't find a better one. cc @remh @olivielpeau

**Comment by olivielpeau** [comment]
Looks good to me!

Don't have any better idea for the parameter name :confused:

**Pull request merged**